src/packages/database/query/index.js

```diff
@@ -164,10 +165,11 @@ class Query<+T: any> extends Promise {
         this.snapshots = this.snapshots
           .filter(([method]) => method !== 'orderBy')
```

Comment:
Also [L#408](https://github.com/nickschot/lux/blob/7acf9de7699fb17e2a5f4da95d184202ee44e704/src/packages/database/query/index.js#L408) should be `return 'orderByRaw';`.